docs/k0sctl-install.md

```diff
@@ -0,0 +1,177 @@
+INFO ==> Running phase: Detect host operating systems 
+INFO [ssh] 10.0.0.1:22: is running Ubuntu 20.10 
+INFO [ssh] 10.0.0.2:22: is running Ubuntu 20.10 
+INFO ==> Running phase: Prepare hosts    
+INFO [ssh] 10.0.0.1:22: installing kubectl     
+INFO ==> Running phase: Gather host facts 
+INFO [ssh] 10.0.0.1:22: discovered bond0 as private interface 
+INFO [ssh] 10.0.0.1:22: discovered 10.12.18.133 as private address 
+INFO ==> Running phase: Validate hosts   
+INFO ==> Running phase: Gather k0s facts 
+INFO ==> Running phase: Download K0s on the hosts 
+INFO [ssh] 10.0.0.2:22: downloading k0s 0.11.0 
+INFO [ssh] 10.0.0.1:22: downloading k0s 0.11.0 
+INFO ==> Running phase: Configure K0s    
+WARN [ssh] 10.0.0.1:22: generating default configuration 
+INFO [ssh] 10.0.0.1:22: validating configuration 
+INFO [ssh] 10.0.0.1:22: configuration was changed 
+INFO ==> Running phase: Initialize K0s Cluster 
+INFO [ssh] 10.0.0.1:22: installing k0s controller 
+INFO [ssh] 10.0.0.1:22: waiting for the k0s service to start 
+INFO [ssh] 10.0.0.1:22: waiting for kubernetes api to respond 
+INFO ==> Running phase: Install workers  
+INFO [ssh] 10.0.0.1:22: generating token       
+INFO [ssh] 10.0.0.2:22: writing join token     
+INFO [ssh] 10.0.0.2:22: installing k0s worker  
+INFO [ssh] 10.0.0.2:22: starting service       
+INFO [ssh] 10.0.0.2:22: waiting for node to become ready 
+INFO ==> Running phase: Disconnect from hosts 
+INFO ==> Finished in 2m2s                
+INFO k0s cluster version 0.11.0 is now installed  
+INFO Tip: To access the cluster you can now fetch the admin kubeconfig using: 
+INFO      k0sctl kubeconfig              
+```
+
+And, -- presto your k0s cluster is up and running
+
+Get kubeconfig:
+```sh
+$ k0sctl kubeconfig > kubeconfig
+$ kubectl get pods --kubeconfig kubeconfig -A
+NAMESPACE     NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-5f6546844f-w8x27   1/1     Running   0          3m50s
+kube-system   calico-node-vd7lx                          1/1     Running   0          3m44s
+kube-system   coredns-5c98d7d4d8-tmrwv                   1/1     Running   0          4m10s
+kube-system   konnectivity-agent-d9xv2                   1/1     Running   0          3m31s
+kube-system   kube-proxy-xp9r9                           1/1     Running   0          4m4s
+kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
+```
+
+### Upgrade k0s version on your setup
+
+The previous example installed k0s v0.11.0-beta.2 on one controller and one worker.
+Let's upgrade the cluster to `v0.11.0` 
+**_NOTE:_**  To get the most recent version please visit the release [page](https://github.com/k0sproject/k0s/tags).
+
+The `k0sctl apply` command will determine the running k0s version on each host and perform an upgrade to the target version defined in the configuration file. `k0sctl apply` will then execute the following:
```

Comment:
```suggestion
There's no dedicated upgrade sub-command in k0sctl, the configuration file describes the desired state of the cluster and when passed to `k0sctl apply`, it will perform a discovery of the current state and do what ever is needed to bring the cluster to the desired state, by for example performing an upgrade.

#### K0sctl cluster upgrade process

The following steps will be performed during a k0sctl cluster upgrade:
```

Maybe just drop the story-line of upgrading the cluster installed in the previous episode and treat this as an independent topic.